config/webpack.common.js

```diff
@@ -122,8 +124,20 @@ module.exports = function (options) {
           use: aotMode ? [
             '@ngtools/webpack'
           ] : [
-              '@angularclass/hmr-loader?pretty=' + !isProd + '&prod=' + isProd,
-              'ts-loader',
+              { loader: 'cache-loader' },
```

Comment:
Since you are not doing anything else in `{ loader: 'cache-loader' }`, you only need `'cache-loader'`. You can remove the braces and `loader`.

Reply:
Changed.